opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -1129,25 +1129,17 @@ assignDynamicWellStatus(data::Wells& wsrpt,
                         const int reportStepIndex) const
 {
     this->loopOwnedWells([this, reportStepIndex, &wsrpt]
-                         (const auto wellID, const Well& well)
+                         ([[maybe_unused]] const auto wellID,
+                          const Well& well)
     {
-        auto& xwel = wsrpt[well.name()]; // data::Wells is a std::map<>
```

Comment:
Why did you move this statement down?

Reply:
@totto82 , if you have time you can recover this line before it gets merged.